templates/index2.html

```diff
@@ -18,55 +18,205 @@
         <br /><br />
         <a href="/logout">Log Out</a>
 
-        <form class="ingredients" action="/search_ingredients" method="POST">
-            <p>Search for Ingredients</p>
-            <input type="text" placeholder="Peanut Butter, Jelly..." />
-            <div class="filters">
-                <p>Filter by:</p>
-                <input type="button" class="filter-button" value="Cuisine" />
-                <input type="button" class="filter-button" value="Alergies" />
+
```

Comment:
This form set up is outdated. A pull with the full index2.html functions for adding has been merged. A pull is needed and editing after pull is to be integrated after testing the pull result first to determine what changes from this pull must be integrated